### ISSUE TITLE
Dokumenttien pysyvä poistaminen roskiksesta

### ISSUE THREAD

**Issue by dezhidki** [TIM-JYU/TIM#3570]
Tällä hetkellä dokumentin poistaminen siirtää sen `roskis`-kansioon säilyttäen oikeudet ja sisällöt. Tämän avulla dokumentin omistaja voi palauttaa sen takaisin näkyviin.
Kuitenkin jos dokumentti on jo roskiksesta, sen poistaminen ei tee tällä hetkellä mitään.

Dokumenteille tulisi tarjota pysyvä poistaminen. Vähintään peruskäyttäjälle on tarjottava "Poista roskiksesta" -painike, joka

- Poistaa dokumentista kaikki oikeudet (myös omistajan)
- Siirtää dokumentin vain ylläpidolle näkyvään kansioon ("tyhjennetyt")

Tämän jälkeen ylläpito voi organisaation tietosuojaselosteen mukaisesti poistaa dokumentteja pysyvästi levyltä massana. Tämän kortin kannalta riittää, että käyttäjälle tarjotaan mahdollisuus poistaa dokumentti kokonaan roskiksesta niin, että kukaan muu kuin ylläpito ei pääse siihen käsiksi.

**Comment by ajlakanen** [comment]
Miten normaali käyttäjä pääsee tällä hetkellä tarkastelemaan poistamiaan dokumentteja? Ainakaan Omat dokumentit -linkin kautta (TIM etusivu) tuollaista näkymää ei tarjota, joten en tiedä mistä tuollainen "poista roskiksesta"-toiminto oikeastaan nykyisellään voisi löytyä. 

Minusta voisi olla hyvä, jos poistamisen yhteydessä tarjottaisiin selkeästi vaihtoehtoja "siirrä roskakoriin" (jota käyttäjä oikeasti pääsisi tarkastelemaan) sekä "poista pysyvästi". 

Toinen vaihtoehto voisi olla tehdä kuten pilvipalveluissa yleensä muutenkin, eli "poista" vie dokumentit aina roskikseen, josta ne poistetaan lopullisesti 30 päivän kuluessa. Tuo mitä "lopullinen poistaminen" tässä yhteydessä tarkoittaa pitänee sitten tosiaan löytyä tai tarvittaessa määritellä tietosuojaselosteessa.

**Comment by dezhidki** [comment]
> Miten normaali käyttäjä pääsee tällä hetkellä tarkastelemaan poistamiaan dokumentteja? Ainakaan Omat dokumentit -linkin kautta (TIM etusivu) tuollaista näkymää ei tarjota, joten en tiedä mistä tuollainen "poista roskiksesta"-toiminto oikeastaan nykyisellään voisi löytyä.

Joo, tällä hetkellä roskikselle ei ole selkeää linkkiä. Pitää itse älytä etusivun kautta klikata Kaikki dokumentit -> Roskis -kansio (eli <https://tim.jyu.fi/view/roskis>). Roskikselle pitäisi olla selkeä linkki siis jossain.

> Minusta voisi olla hyvä, jos poistamisen yhteydessä tarjottaisiin selkeästi vaihtoehtoja "siirrä roskakoriin" (jota käyttäjä oikeasti pääsisi tarkastelemaan) sekä "poista pysyvästi".

Juu, tuo on hyvä ajatus.

**Comment by ajlakanen** [comment]
Tähän vielä lisäyksenä, että nykyisen Delete document UI:n teksti antaa oikeastaan hyvin selkeästi ymmärtää, että poisto on todellakin lopullinen. 

    Are you sure you want to delete this document? Deletion cannot be undone!
